draft-ietf-secevent-http-push.xml

```diff
@@ -314,7 +304,7 @@ Content-Type: application/json
                     <c>setType</c><c>An unexpected Event type was received.</c>
                     <c>setParse</c><c>Invalid structure was encountered such as an
                         inability to parse or an incomplete set of Event claims.</c>
-                    <c>setData</c><c>SET event claims incomplete or invalid.</c>
```

Comment:
I had interpreted this error code as applying to the situation where the claims within the Event Payload are incomplete or invalid. However, it looks like `setParse` is intended to cover that scenario. Perhaps `setClaims` would be a clearer name for this error code?

Examples would help here.

Reply:
We need to work on the definition for `setParse` and `setData` then; there seems to be a lot of overlap in their scopes.

Reply:
It's not clear to me that we need to refine the definitions before publishing -03.  I'd rather we have broader discussion before tweaking the definitions - discussions that we can have in Bangkok.  My change was simply to eliminate one of the (several) uses of the redundant and confusing terminology "SET event".

Do you concur?

Reply:
Yes, I'm fine with publishing -03 with the error codes as they are in this pull request.

Reply:
Thanks.  I'm going to proceed then so I can turn my attention back to the poll draft.